### ISSUE TITLE
Sprout, flower, etc icons position with Small Job Icon and Name

### ISSUE THREAD

**Issue by Evolutious** [shdwp/xivPartyIcons#36]
Greetings! First off, Thank you! for adding the support to show the player status icons (sprout, flower, etc) with the job icon when using the small icon and name option!

Using the following screenshot example for my below comments: https://gyazo.com/aef618c7ab90957c9858de11f0da9b40 

It looks like you are attaching your job icon to the left of the status icon, which has a side effect of causing a large gap between the job icon and the player name when there is no status icon (you can see this in the above screenshot comparing the RPR with their returning player icon and the CUL player above them taking note on the gap between the icon and their actual player name). Is there any way you can either attach the job icon to the player name or on the right side of the status icon?

Second; is it possible to not cause the status icon to become very small and instead maintain its original size? See screenshot comparing the original size (top nameplates) and the small size with your nameplate hook enabled (lower nameplates) to get an idea.

Lastly; is it possible to expand this to show all status icons in game. In the above image comparison, you can see the upper player has a group status icon when using the game default nameplate but it is not displayed when using your adjusted nameplate. I can confirm that this is the case for the group icon in the image as well as others like AFK and when in queue for an instance.

Again, thank you!

**Comment by catgirlfighter** [comment]
That's because that's not a status icon, it's a chat icon that normally used in mentor chat.

**Comment by Evolutious** [comment]
would this be the same for the indicators like when you are in a party, in an instance queue or watching a cutscene? Those also do not show (as additional examples). If it were possible to just take whatever is in that icon slot and maintain it, that would be perfect as i know you dont see in an instance queue icon concurrently with your mentor icon; for example.

if it isn't possible, that is cool, but if it can be accomplished, it would greatly increase the usability of this plugin.

**Comment by catgirlfighter** [comment]
Yeah, author doesn't add those icons because they can't be added to the character name.
Job icons are actual replacement of status icons.
I've suggest a way to show rather important status icons INSTEAD of job icons (since they're usually temporary, it shouldn't be a problem), but author have refused it for one reason or another.

**Comment by catgirlfighter** [comment]
In my suggestion that was everything except _ignorables, which included "Party Leader", "Party Member" and all adventurer statuses (mentor and etc). That does mean that in dungeon you could see "busy" and "RP" statuses, but again, people don't normally use them in dungeons, and it's useful to see them in overworld, so maybe an option could work.
